cmd/proxy/shutdown/signals.go

```diff
@@ -0,0 +1,15 @@
+//go:build unix
+
+package shutdown
+
+import (
+	"os"
+	"syscall"
+)
+
+// GetSignals returns the appropriate signals to catch for a clean shutdown, dependent on the OS.
+//
+// On Unix-like operating systems, it is important to catch SIGTERM in addition to SIGINT.
+func GetSignals() []os.Signal {
```

Comment:
Does this really need to be a package? Could it not instead go into `cmd/proxy/signals_unix.go`? In that case it would not need to be exposed. Better yet, it could just be a var.

Reply:
Sure thing, if that's how you want it. I was just attempting to keep the `main` package clean, since it was currently only a single `main.go` file. I've always avoided putting OS-specific things into `main`.

If we just want to avoid exporting, another option would be to put it into `internal/shutdown`, instead of `cmd/proxy/shutdown`.

Reply:
Also... I'm a bit confused. Are you a maintainer? You don't seem to have any previous contributions to this repo...

Reply:
No, I am not a maintainer, I am just helping out with some PRs.

Reply:
Hello, 
@nrwiersma is helping us out as we are looking for new maintainers.

https://github.com/gomods/athens/issues/1734